core/src/main/java/com/yahoo/oak/InternalOakBasics.java

```diff
@@ -184,7 +188,8 @@ V replace(K key, V value, OakTransformer<V> valueDeserializeTransformer) {
         throw new RuntimeException("replace failed: reached retry limit (1024).");
     }
 
-    boolean replace(K key, V oldValue, V newValue, OakTransformer<V> valueDeserializeTransformer) {
+    boolean replace(K key, V oldValue, V newValue, OakTransformer<V> valueDeserializeTransformer) 
+            throws DeletedMemoryAccessException {
```

Comment:
It is unclear, why `replace()` method above retries upon exception and this one redirects the exception... ?